pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -137,60 +139,80 @@ social:
 
   %br/
 
-  .row
-    .col-xs-6.col-sm-4
-      %img{src: "/images/fill-600x400/highlight-girl-hawaii.jpg", style: "width: 100%;"}
+  -if @language == 'en'
+    .row
+      .col-md-8
+        %a.twitter-moment{:href => "https://twitter.com/i/moments/803745761305296896?limit=4"} The Hour of Code 2016
+        %script{:async => "", :charset => "utf-8", :src => "//platform.twitter.com/widgets.js"}
 
-    .col-xs-6.col-sm-4
-      %img{src: "/images/fill-600x400/highlight-robot.jpg", style: "width: 100%;"}
+      .col-md-3.highlights-column
+        %h2.white-text Highlights from previous years
+        %div.highlight-tiles
+          - DB[:cdo_highlights].limit(100).reverse_each do |item|
+            - if item[:id] == 16 || item[:id] == 19 || item[:id] == 45
```

Comment:
Why only these 3 ids? Do we ignore others? If so, why not query `DB[:cdo_highlights].where(id: [16, 19, 45])...`?